tools/src/test/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommandArgsTest.java

```diff
@@ -29,7 +29,7 @@
 
 @Timeout(60)
 public class ReassignPartitionsCommandArgsTest {
-    public static final String MISSING_BOOTSTRAP_SERVER_MSG = "Please specify --bootstrap-server";
+    public static final String MISSING_BOOTSTRAP_SERVER_MSG = "Please specify either --bootstrap-server";
```

Comment:
why we need this change?

Reply:
I change the error message in validate method, thus these error message should be changed
```java
if (opts.options.has(opts.bootstrapServerOpt) && opts.options.has(opts.bootstrapControllerOpt)) 
    CommandLineUtils.printUsageAndExit(opts.parser, "Please don't specify both --bootstrap-server and --bootstrap-controller");
else if (!opts.options.has(opts.bootstrapServerOpt) && !opts.options.has(opts.bootstrapControllerOpt))
    CommandLineUtils.printUsageAndExit(opts.parser, "Please specify either --bootstrap-server or --bootstrap-controller");
else 
    bootstrapOpt = opts.options.has(opts.bootstrapServerOpt) ? opts.bootstrapServerOpt : opts.bootstrapControllerOpt;
```